cvat-core/src/quality-settings.ts

```diff
@@ -60,7 +63,11 @@ export default class QualitySettings {
     }
 
     get task(): number {
-        return this.#task;
+        return this.#task_id;
+    }
+
+    get project(): number {
```

Comment:
Looks like number | null is correct for both task and project getters
```suggestion
    get project(): number | null {
```